### DISCUSSION TITLE
export pdf on addroid

### DISCUSSION THREAD

**Discussion by arias007** [l1xnan/obsidian-better-export-pdf#171] [Ideas]
please add a function of export pdf on addroid,it will be popular,thank you

**Comment by arias007** [comment]
I hope add a picture of alipay on the page of buy me a coffee if you have alipay account,I really like your plugin

**Reply by l1xnan** [reply]
Thank you for your recognition! About Android support, it may not be feasible to implement because the plugin relies on Obsidian's PDF export API, which in turn depends on [Electron](https://www.electronjs.org/), available only on desktop platforms. There is no corresponding API for Android.

**Comment by arias007** [comment]
Thank you for your reply,I like your plugin and thank you for your plugin.



---Original---
From: ***@***.***&gt;
Date: Fri, Apr 12, 2024 13:50 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [l1xnan/obsidian-better-export-pdf] export pdf on addroid(Discussion #171)




 
Thank you for your recognition! About Android support, it may not be feasible to implement because the plugin relies on Obsidian's PDF export API, which in turn depends on Electron, available only on desktop platforms. There is no corresponding API for Android.
